source/plugins/go/input/lib/cadvisor.go

```diff
@@ -267,12 +268,15 @@ func GetMetricsHelper(metricInfo map[string]interface{}, winNode map[string]stri
 		} else {
 			hostName = GetHostname()
 		}
-		operatingSystem = "Linux"
+		if osType != "" && strings.EqualFold(osType, "windows") {
+			operatingSystem = "Windows"
+		} else {
+			operatingSystem = "Linux"
+		}
 	}
 
-	osType := strings.TrimSpace(os.Getenv("OS_TYPE"))
 	// Checking if we are in windows daemonset and sending only a few metrics that are needed for MDM
-	if osType != "" && strings.EqualFold(osType, "windows") {
+	if osType != "" && strings.EqualFold(osType, "windows") && !IsAADMSIAuthMode() {
```

Comment:
not sure I understand this. Windows fluent-bit input plugins are only for MSI mode. In that case what are we collecting here?

Reply:
I think this is dead code. It is present in the ruby plugins for differentiating what and how RS collects the data. Do you think I should just remove this code? @pfrcks

Reply:
yeah lets remove. just double check that this does not have any other effects

Reply:
I was thinking lets keep it as is for now and then later we can remove when everything is set up and we know what default exactly looks like? wdyt?

Reply:
DEAD CODE - will be removed later